src/oaklib/converters/obo_graph_to_fhir_converter.py

```diff
@@ -97,47 +99,93 @@ def dump(
         self,
         source: GraphDocument,
         target: str = None,
+        code_system_id: str = None,
+        code_system_url: str = None,
         include_all_predicates: bool = True,
+        native_uri_stems: List[str] = None,
+        use_curies_native_concepts: bool = False,
+        use_curies_foreign_concepts: bool = True,
+        predicate_period_replacement: bool = False,
         **kwargs,
     ) -> None:
         """
         Dump an OBO Graph Document to a FHIR CodeSystem.
 
-        By default, only IS_A predicates are converted to ConceptProperties. To override this,
-        specify ``include_all_predicates=True``.
-
-        :param source:
-        :param target:
-        :param include_all_predicates: include the maximal amount of predicates
+        :param source: Source serialization.
+        :param target: Target serialization.
+        :param code_system_id: The code system ID to use for identification on the server uploaded to.
```

Comment:
**Docs**
Added documentation for params to `OboGraphToFHIRConverter`.

@cmungall You mentioned
> But for now the SOT docs can be the docstring in the fhir converter class

I'm not sure what SOT means; I tried googling it but couldn't find.

Reply:
https://en.wikipedia.org/wiki/Single_source_of_truth